packages/mock/src/fulcio/handler.ts

```diff
@@ -22,6 +22,9 @@ import type { CA } from './ca';
 
 const CREATE_SIGNING_CERT_PATH = '/api/v2/signingCert';
 const DEFAULT_SUBJECT = 'NO-SUBJECT';
+const DEFAULT_ISSUER = 'https://token.actions.githubusercontent.com';
```

Comment:
Thoughts on setting this to some obviously fake default value? Feels a bit weird having github.com's issuer in the mock :)

Reply:
Good call.